tests/automation/linked_device_requests.spec.ts

```diff
@@ -11,68 +9,81 @@ import {
   waitForTextMessage,
 } from './utilities/utils';
 
-sessionTestTwoWindows('Accept request syncs', async ([windowA, windowB]) => {
-  const [userA, userB] = await Promise.all([
-    newUser(windowA, 'Alice'),
-    newUser(windowB, 'Bob'),
-  ]);
-  const [windowC] = await linkedDevice(userB.recoveryPhrase);
+sessionTestThreeWindowsWithTwoLinked(
+  'Accept request syncs',
+  ['Alice', 'Bob'],
+  async (
+    { windowsLinked: [windowA, windowB], otherWindow },
+    { userLinked, otherUser },
+  ) => {
+    const testMessage = `${otherUser.userName} sending message request to ${userLinked.userName}`;
```

Comment:
Nor otherUser, don't understand who otherUser is

Reply:
the same logic applies,
userA and userB doesn't tell you which one is the one linked in two windows and which one is the other one.
Most of the time userA is the one linked, but sometimes userB is the one linked.
Having it explicit makes it easier to write meaninful tests

Reply:
![image](https://github.com/oxen-io/session-playwright/assets/1544279/5140c936-2496-4036-8033-81ee47461513)